### ISSUE TITLE
update python wheel docker CI setup

### ISSUE THREAD

**Issue by lkeegan** [spatial-model-editor/spatial-model-editor#556]
- the PyPy docker image for building wheels is getting merged into the pypa one (https://github.com/pypa/manylinux/issues/1099)
- cibuildwheel will soon release v2 with support for this (https://github.com/pypa/cibuildwheel/pull/671)

Once this is done:

- we can archive https://github.com/spatial-model-editor/sme_manylinux2010-pypy_x86_64
- use https://github.com/spatial-model-editor/sme_manylinux2010_x86_64 to build all linux wheels

**Comment by lkeegan** [comment]
resolved by #605
